syft/frameworks/torch/mpc/fss.py

```diff
@@ -282,6 +322,19 @@ class DPF:
 
     @staticmethod
     def keygen(n_values=1):
+        return dpf.keygen(n_values=n_values)
+
+    @staticmethod
+    def eval(b, x, k_b):
+        # x = x.astype(np.uint64)
+        original_shape = x.shape
+        x = x.reshape(-1)
+        flat_result = dpf.eval(b, x, k_b)
+        return flat_result.astype(np.int64).reshape(original_shape)
+        # return result
+
+    @staticmethod
+    def py_keygen(n_values=1):
```

Comment:
You can also rm py_keygen and py_eval : they are still accessible in the commit history and the cuda version don't use it :)

Reply:
Yes, I was keeping that for CUDA :) I cleaned this stuff and the comments!